themes/scss/map/dashboard-info.scss

```diff
@@ -157,15 +155,15 @@
 }
 .CDB-Dashboard-infoSnapshotsListTitle {
   margin-bottom: 8px;
-  font-size: $sFontSize-smaller;
+  font-size: $sFontSize-small;
   font-weight: $sFontWeight-semibold;
-  line-height: $sLineHeight-smaller;
+  line-height: $sLineHeight-small;
 }
 .CDB-Dashboard-infoSnapshotsListDate {
   color: #636D72;
-  font-size: $sFontSize-smaller;
+  font-size: $sFontSize-small;
   font-weight: $sFontWeight-lighter;
-  line-height: $sLineHeight-smaller;
+  line-height: $sLineHeight-small;
```

Comment:
Name of variable `sLineHeight-small` should be written in all lowercase letters with hyphens instead of underscores